lara-typescript/src/plugin-api/types.ts

```diff
@@ -82,6 +82,14 @@ export interface IEmbeddableRuntimeContext {
    interactive status, including click to play state.
    ****************************************************************************/
   clickToPlayId: string | null;
```

Comment:
I think this can be removed from the V3 api. We haven't released a version with it, and now plugins can use onClickToPlayStarted method instead.

Reply:
Okay. I didn't know if I'd have time to actually update LARA Sharing Plugin and I didn't want to block the whole release process.

Actually, now it seems it's even worse. I think LARA Sharing Plugin doesn't use click-to-play at all. Something has been changed and I'm not sure if it's intentional or not. Probably not. The code performs monitoring, but its result is just never used. There are no tests. And the local setup of this whole thing is pretty painful. So, I can definitely remove this id and perhaps we should assume that Sharing Plugin should be fixed (as it's probably broken at the moment).

I can definitely take a look at it, although if Doug or Noah have some time, it might be more efficient for them. They understand this code better and probably they have a bunch of local tests available.

Reply:
OK, that sounds like a good reason to leave this in for the time being since I'm not sure when we'll get to fixing the sharing plugin.